meta-lmp-base/recipes-samples/images/lmp-image-common.inc

```diff
@@ -43,6 +43,9 @@ CORE_IMAGE_BASE_INSTALL:append:sota = " \
     ostree-recovery-initramfs \
 "
 
+# Common features
+require ${@bb.utils.contains('COMBINED_FEATURES', 'alsa', 'lmp-feature-alsa.inc', '', d)}
```

Comment:
I think the correct would be distro features instead, combined is a way for yocto to match what is available in the hardware (matching distro with machine features).

Reply:
I didn't want to alter `packagegroup-base.bb`. But yeah, distro is the better place, as it was from the beginning. Ok, fixing.

Reply:
Also, I re-thought using the full alsa-utls package, it may be needed for debug build only, so I moved alsa-utils to the feature-debug, adding only `alsa-utils-aplay` (which includes `arecord` as well) to make alsa subsystem usable in LmP images.